packages/commerce-sdk-react/src/hooks/ShopperPromotions/query.test.ts

```diff
@@ -28,8 +28,8 @@ const OPTIONS = {parameters: {campaignId: 'campaignId', ids: 'a,b'}}
 type TestMap = {[K in keyof Queries]: NonNullable<ReturnType<Queries[K]>['data']>}
 // This is an object rather than an array to more easily ensure we cover all hooks
 const testMap: TestMap = {
-    usePromotions: {count: 0, data: [], total: 0},
-    usePromotionsForCampaign: {count: 0, data: [], total: 0}
+    usePromotions: {limit: 0, data: [], total: 0},
+    usePromotionsForCampaign: {limit: 0, data: [], total: 0}
```

Comment:
Was there a breaking change in the API? count --> limit?

Reply:
yep! and we are bumping a major for commerce-sdk-react as well